app/models/bot.rb

```diff
@@ -1,3 +1,5 @@
+require 'indico'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.